src/renderers/testing/stack/ReactTestMount.js

```diff
@@ -138,6 +139,20 @@ ReactTestInstance.prototype.unmount = function(nextElement) {
   });
   this._component = null;
 };
+ReactTestInstance.prototype.toTree = function() {
+  const component = this._component._renderedComponent;
+  const element = component._currentElement;
+  // not using `children`, but I don't want to rewrite without destructuring
+  // eslint-disable-next-line no-unused-vars
+  const { children, ...propsWithoutChildren } = element.props;
+  return {
+    nodeType: ReactNodeTypes.getType(element),
```

Comment:
`ReactNodeTypes` returns an integer representing the type, but in the spec we define `nodeType` as a string. Should we update the spec and standardize the numbering, e.g., `0` always refers to a host node?